docs/sphinx/tutorial/allocators.rst

```diff
@@ -13,14 +13,19 @@ All :class:`umpire::Allocator` s are created and managed by Umpire's
 :class:`umpire::ResourceManager`. To get an Allocator, you need to ask for one: 
 
 .. literalinclude:: ../../../examples/tutorial/tut_allocator.cpp
-                    :lines: 14-16
+   :start-after: _umpire_tut_get_allocator_start
+   :end-before: _umpire_tut_get_allocator_end
+   :language: C++
 
 Once you have an :class:`umpire::Allocator` you can use it to allocate and deallocate memory:
 
 .. literalinclude:: ../../../examples/tutorial/tut_allocator.cpp
-                    :lines: 18-24
+   :start-after: _umpire_tut_de_allocate_start
```

Comment:
This section of the code shows getting a size, allocating memory, printing out what you have, and then deallocating. What do you think about shaving it down to just allocate and deallocating? (Or perhaps just what the original code snippet was - the allocation, print statement, and deallocating). Do you see what I mean?

Reply:
Similar question for the Dynamic Pools Tutorial page

Reply:
Thanks @kab163, can you enter this as a suggestion (lille +/- box) at the top?